backend/src/api.ts

```diff
@@ -0,0 +1,306 @@
+import bodyParser from "body-parser";
+import cors from "cors";
+import express from "express";
+import multer from "multer";
+import { isJson } from "./utils/index.js";
+
+import {
+  cidToString,
+  IPLDNodeData,
+  OffchainMetadata,
+} from "@autonomys/auto-drive";
+import { decode } from "@ipld/dag-pb";
+import "dotenv/config.js";
+import { FolderTreeSchema, NodeWithMetadata } from "./models/index.js";
+import { transactionResultsRepository } from "./repositories/index.js";
+import { handleAuth } from "./services/authManager/express.js";
+import {
+  FilesUseCases,
+  MetadataUseCases,
+  NodesUseCases,
+  TransactionResultsUseCases,
+} from "./useCases/index.js";
+
+const setContentTypeHeaders = (
+  res: express.Response,
+  metadata: OffchainMetadata
+) => {
+  res.set(
+    "Content-Type",
+    (metadata.type === "file" && metadata.mimeType) ||
+      "application/octet-stream"
+  );
+  if (metadata.name) {
+    console.log(
+      `Setting Content-Disposition to attachment with filename: ${metadata.name}`
+    );
+    res.set("Content-Disposition", `attachment; filename="${metadata.name}"`);
+  }
+};
+
+const createServer = async () => {
+  const app = express();
+  const port = Number(process.env.PORT) || 3000;
+
+  // Increase the limit to 10MB (adjust as needed)
+  const requestSizeLimit = process.env.REQUEST_SIZE_LIMIT || "10mb";
```

Comment:
Can we add a .env or .env.sample (depending on if you also have .env that are true secret) so we have, somewhere in the repo, a list of all the .env defined in the code?